test/renate/shift/test_detector.py

```diff
@@ -0,0 +1,20 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+import pytest
+import torch
+
+from renate.shift.detector import ShiftDetectorWithFeatureExtractor
+
+
+@pytest.mark.parametrize(
+    "dataset",
+    [
+        torch.utils.data.TensorDataset(torch.randn(20, 2)),
+        torch.utils.data.TensorDataset(torch.randn(20, 2), torch.randint(0, 10, size=(20,))),
+    ],
+)
+@pytest.mark.parametrize("feature_extractor", [torch.nn.Linear(2, 4)])
```

Comment:
can we check the output of the feature extractor? for example by using a model initialized ad-hoc (e.g., all-ones)

Reply:
Will do.